windows-nsis/INSTALL-win32.txt

```diff
@@ -0,0 +1,62 @@
+GENERAL QUICKSTART FOR WINDOWS
+
+OpenVPN requires a configuration file and key/certificate files. You should 
+obtain these and save them to OpenVPN's configuration directory, usually 
+C:\Program Files\OpenVPN\config. Starting with OpenVPN 2.4-alpha1 you can also 
+put user-specific configurations to %USERPROFILE%\OpenVPN\config when using 
+OpenVPN GUI.
```

Comment:
Scanning both seems quite easy, based on a cursory look at [openvpn_config.c](https://github.com/OpenVPN/openvpn-gui/blob/master/openvpn_config.c#L220). The question then becomes: do we _want_ to have two places where users can put user-specific config files? If we remove either one later we will break user configs.